lib/web_ui/lib/src/engine/canvaskit/image.dart

```diff
@@ -350,6 +350,9 @@ class CkImage implements ui.Image, StackTraceDebugger {
     }
   }
 
+  @override
+  int get approximateBytesUsed => width * height * 4;
```

Comment:
We should ask CanvasKit what the size is, it knows a much better value than this.

Reply:
It does not appear CanvasKit actually exposes this on `Picture` or `Image`, so this would require a separate change to the skia repo to add this (right now the `Picture` implementation for CanvasKit just returns 0 for `approximateBytesUsed`)

Reply:
I added `Picture.approximateBytesUsed` to CanvasKit: https://github.com/google/skia/commit/46d3564c1e1b1814a12c6cdeeda5eccf636a216f